iothub/device/src/Edge/EdgeModuleClientHelper.cs

```diff
@@ -31,29 +31,7 @@ internal class EdgeModuleClientFactory
         private const string IothubConnectionstringVariableName = "IotHubConnectionString";
         private const string EdgeCaCertificateFileVariableName = "EdgeModuleCACertificateFile";
 
-        private readonly ITrustBundleProvider _trustBundleProvider;
-        private readonly IotHubClientOptions _options;
-
-        /// <summary>
-        /// Initializes a new instance of the class with transport settings.
-        /// </summary>
-        /// <param name="trustBundleProvider">Provider implementation to get trusted bundle for certificate validation.</param>
-        /// <param name="options">The options that allow configuration of the module client instance during initialization.</param>
-        public EdgeModuleClientFactory(ITrustBundleProvider trustBundleProvider, IotHubClientOptions options = default)
-        {
-            _trustBundleProvider = trustBundleProvider ?? throw new ArgumentNullException(nameof(trustBundleProvider));
-            _options = options ?? new();
-        }
-
-        /// <summary>
-        /// Creates an instance based on environment.
-        /// </summary>
-        public Task<IotHubModuleClient> CreateAsync()
-        {
-            return CreateInternalClientFromEnvironmentAsync();
-        }
-
-        private async Task<IotHubModuleClient> CreateInternalClientFromEnvironmentAsync()
+        internal static IotHubConnectionCredentials CreateIotHubConnectionCredentialsFromEnvironment()
```

Comment:
I'm looking at cleaning this class up a bit further, but I've opened the PR up for comments in the meanwhile.